framework/include/dm/dm_connectivity.h

```diff
@@ -35,11 +35,12 @@ extern "C"
 {
 #endif
 
-#ifdef DM_API_DEBUG
-#define dmdbg(format, ...) printf("DM_API (%s): " format, __FUNCTION__, ##__VA_ARGS__)
-#else
-#define dmdbg(a, ...) (void)0
-#endif
+typedef struct dm_scan_info_s {
```

Comment:
According to guide, below style is right.
struct dm_scan_info_s {
}
typedef struct dm_scan_info_s dm_scan_info_t